hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -529,15 +529,23 @@
       name: chip_rv_dm_perform_debug
       desc: '''Verify that Ibex enters debug mode when triggered through JTAG
 
-            - While Ibex is executing, trigger a debug request by programming `dmcontrol.haltreq`
-              via JTAG.
-            - Ibex steps into interactive debug to execute instructions in SRAM.
-            - Load instruction data into the SRAM via SW. X-ref'ed with chip_sw_sram_execution.
-            - Program the program buffer via JTAG with code to let Ibex jump to the SRAM to
-              execute the instructions.
-            - Verfy that Ibex can fetch and execute the instructions from the SRAM correctly.
-            '''
-      milestone: V2
+            This test must be run with mask ROM.  Mask ROM is responsible for enabling debug ROM
+            access and must be used to ensure the epmp configurations are done correctly.
```

Comment:
I think we probably want to move this test-point to the `mask_rom_testplan.hjson` [here](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/mask_rom/data/mask_rom_testplan.hjson) (which gets linked to this testplan at the top via the include statements).